src/pages/find-my-combos.tsx

```diff
@@ -190,6 +210,27 @@ const FindMyCombos = () => {
               Paste your decklist
             </div>
           )}
+
```

Comment:
I'd put a
```
<p>
  or
</p>
```
in between the two inputs so users don't think they need both. Style it a bit, if it looks bad.